bin/masspay.py

```diff
@@ -193,6 +193,11 @@ def post_back_to_gratipay():
     except KeyError:
         gratipay_base_url = 'https://gratipay.com'
 
+    nmasspays = int(requests.get(gratipay_base_url + '/dashboard/nmasspays').text())
+    if nmasspays < 10:
```

Comment:
In case that there was no more than 10 teams to pay out to. Or is that impossible?